### DISCUSSION TITLE
How to embed a drawer toggle inside of my nav menu list item?

### DISCUSSION THREAD

**Discussion by victornasar** [saadeghi/daisyui#2614] [Q&A] [answered]
I want the user to click the button in the drawer-side, then the sidebar closes. I'm not sure how to make the second label inside of DashboardModal wrap around my btn. 

Link: https://stackblitz.com/edit/daisyui-nextjs-tq2uvj?file=app%2FdashboardModal.tsx

**Comment by saadeghi** [comment, marked as answer]
Any `<label htmlFor="dash-drawer">` opens/closes the drawer so if you want a button in sidebar to close it, you can use that.  
Also it looks like you want it to look like the menu item, in that case use it in a `<li>`

https://stackblitz.com/edit/daisyui-nextjs-9lwi4w?file=app%2FdashboardModal.tsx

Let me know if you have a question.

**Comment by victornasar** [comment]
YOU SAVED ME SO MUCH TIME!!!! THANK YOU SO MUCH!!!! I LOVE YOUR PRODUCT! 

Keep it up!

**Reply by victornasar** [reply]
Now I have a new issue. I want to embed a next/link container inside my label instead of just having a button that says close. I want to be able to route to another page while closing the sidebar at the same time.

**Reply by saadeghi** [reply]
In that case, don't use `<label>`. Bind a variable to the drawer's hidden checkbox, then set that variable to false when the menu item is clicked.
